server/src/test/java/com/paypal/selion/grid/servlets/GridStatisticsTest.java

```diff
@@ -0,0 +1,77 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2016 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.grid.servlets;
+
+import static org.powermock.api.mockito.PowerMockito.spy;
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+import static org.powermock.api.support.membermodification.MemberModifier.stub;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openqa.grid.internal.Registry;
+import org.openqa.grid.web.servlet.RegistryBasedServlet;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.paypal.selion.pojos.BrowserInformationCache;
+import com.paypal.selion.pojos.BrowserStatisticsCollection;
+import com.paypal.selion.pojos.BrowserStatisticsCollection.BrowserStatistics;
+
+/**
+ * Tests for GridStatistics servlet
+ */
+@PrepareForTest({ GridStatistics.class, BrowserInformationCache.class, RegistryBasedServlet.class, Registry.class })
```

Comment:
Only `GridStatistics.class` needs to be specified on this line.  I'll update the PR.